src/articulation.ts

```diff
@@ -285,7 +285,7 @@ export class Articulation extends Modifier {
         if (position) artic.position = Modifier.PositionString[position];
         return builder.getFactory().Articulation(artic);
       })
-      .map((artic) => note.addModifier(0, artic));
+      .map((artic) => note.addModifier(artic, 0));
```

Comment:
Remove the '0' this is the default value.

Reply:
I'm going to leave the 0s there for now, to be explicit.